week-2-practise/level-2/4-slice.js

```diff
@@ -5,7 +5,7 @@
 // - remove the item at the specified index
 
 function remove(arr, index) {
-  return; // complete this statement
+  return arr.slice(0, index).concat(arr.slice(index + 1, arr.length)); // complete this statement
```

Comment:
Good. As Daniel said in the last class after looking up the docs online, you don't need the second argument to `slice` when you want it to take all the elements to the end of the array. So `arr.slice(index + 1, arr.length)` can just be written as `arr.slice(index + 1)`.